tests/wicked/before_test.pm

```diff
@@ -54,6 +54,21 @@ sub run {
         systemctl 'start dhcpd.service';
     } else {
         # Common SUT Configuration
+        if (get_var('WICKED_SOURCES')) {
+            zypper_call('--quiet in -t pattern Basis-Devel');
+            zypper_call('--quiet in automake autoconf libtool libnl-devel libnl3-devel libiw-devel dbus-1-devel pkg-config libgcrypt-devel systemd-devel git');
+            my $repo_url = get_var('WICKED_SOURCES');
+            if ($repo_url =~ /\#/) {
+                my ($repo_url, $branch) = ($repo_url =~ /(.*)\#(.*)/);
+                assert_script_run('git clone ' . $repo_url);
+                assert_script_run("git checkout -b $branch origin/$branch");
+            }
+            else {
+                assert_script_run('git clone ' . $repo_url);
+            }
+            assert_script_run('cd ./wicked ; ./autogen.sh ', timeout => 600);
+            assert_script_run('make ; make install',         timeout => 600);
```

Comment:
Good idea to run openQA tests against version from git repo. Maybe we can make it as function in the future, if we ant test more packages  like this.

Reply:
yes and no. idea to having ability to run custom package version in openQA is "flying around" for ages but problem here that to really achieve we basically have too options : 
1. invent OBS inside openQA :) 
2. do a proper integration of openQA and OBS so you can "ask"  OBS to build you certain sources . 
Because what I am doing here is custom build script for certain package and there is no way to do it generic - each package has it's own list of dependencies needs to be installed and even it's own bash scripts which needs to be run ...